<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kasir Sederhana</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBoxProduk">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>-select-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tahu (Rp.2000)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tempe (Rp.4000)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ikan (Rp.10,000)</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Product</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditJumlah">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantity</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Discount</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxDiskon">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>70%</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add To Cart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClr">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelTotal">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total : Rp.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Lalu Muhammad Noval A. - F1D022056</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>355</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
